lib/moog-require.js

```diff
@@ -76,7 +76,8 @@ module.exports = function(options) {
       projectLevelPath = matches[0] ? path.normalize(matches[0]) : projectLevelPath;
     }
     if (fs.existsSync(projectLevelPath)) {
-      projectLevelDefinition = importFresh(resolveFrom(path.dirname(self.root.filename), projectLevelPath));
```

Comment:
OK now here we could be in trouble. `importFresh` was used to make sure that in a multisite context, every site gets its own safe-to-modify copy of each import. I'm not sure how much that still matters or if we can work around it, but at one point this was critical to get multisite to work properly.

Reply:
Fixed it with `const importFresh = moduleName => import(`${moduleName}?${Date.now()}`);`

Thanks to https://github.com/sindresorhus/import-fresh?tab=readme-ov-file#esm